packages/web/docs/src/components/cli-errors.tsx

```diff
@@ -0,0 +1,381 @@
+import { ReactElement } from 'react';
+import type { GetStaticProps } from 'next';
+
+type CLIError = {
+  code: string;
+  title: string;
+  example: string;
+  exampleOutput: string;
+  fix: string;
+};
+
+export function ErrorDetails(props: CLIError): ReactElement {
+  return (
+    <>
+      <h3>
+        HC{props.code} "{props.title}"
+      </h3>
+      <div>
+        Example: <code>${props.example}</code>
+      </div>
+      <pre>"{props.exampleOutput}"</pre>
+      <h4>Suggested Fix</h4>
+      <p>{props.fix}</p>
+    </>
+  );
+}
+
+export async function getErrorDescriptions(): Promise<CLIError[]> {
+  const examples: Array<{
+    instance: { code: string; message: string; name: string };
+    example: string;
+    fix: string;
+  }> = [
+    // @note: cannot import the actual errors due to missing internal node modules.
+    //   But structure it as if we can for the future.
+
+    // { instance: new errors.APIError('Something went wrong', '12345'), ...}
```

Comment:
I tried to structure this so that in the future, we can easily swap the `instance` for a real example instance like this.
But I could not get the docs to build if importing the errors at this time. I think the errors code would need broken out because oclif has a lot of dependencies that nextjs doesnt support normally